src/lib/resource/sections/context-rep-details.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+module.exports = {
+    section: {
+        schema: {
+            $schema: 'http://json-schema.org/draft-07/schema#',
+            type: 'object',
+            additionalProperties: false,
+            properties: {
+                'context-rep-details': {
+                    title: 'Your details',
+                    description:
+                        '<p class="govuk-body">We\'re going to ask for some details about you.</p><p class="govuk-body">As you\'re the victim\'s representative, we\'ll use these details to contact you about this claim.</p>'
```

Comment:
@neil-stephen-mcgonigle can we change "claim" to "application" but only if it's no hassle? I missed this but "claim" is inconsistent with the rest of the application. I'll get Graham to update the prototype if you're ok to change it